ArduCopter/Copter.h

```diff
@@ -323,7 +323,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
             uint8_t initialised             : 1; // 13      // true once the init_ardupilot function has completed.  Extended status to GCS is not sent until this completes
             uint8_t land_complete_maybe     : 1; // 14      // true if we may have landed (less strict version of land_complete)
             uint8_t throttle_zero           : 1; // 15      // true if the throttle stick is at zero, debounced, determines if pilot intends shut-down when not using motor interlock
-            uint8_t system_time_set         : 1; // 16      // true if the system time has been set from the GPS
+            uint8_t system_time_set_unused  : 1; // 16      // true if the system time has been set from the GPS
```

Comment:
Is this used ?

Reply:
no, and its kept to keep the bit numbering